FEXCore/Source/Interface/Core/OpcodeDispatcher.h

```diff
@@ -112,7 +112,15 @@ class OpDispatchBuilder final : public IREmitter {
     // Changes get stored out by CalculateDeferredFlags.
     CachedNZCV = nullptr;
     CFInverted = CFInvertedABI;
-    FlushRegisterCache();
```

Comment:
I think keeping this then setting MMXState directly without using ChgStateMMX_X87 might be cleaner (can also drop the force param)

Reply:
It seems good to me to differentiate just a cache reset to an actual state transition here, and as a counterpoint, if some change to x87 transition stopped it from flushing the register cache then everything would be broken here.

Reply:
Yeah, I think it goes both ways. Not totally convinced about this tbh but I don't think it's that relevant. I can change it if you prefer.